nixpkgs/mkshell.nix

```diff
@@ -0,0 +1,24 @@
+{ nixpkgs ? import <nixpkgs> {}, ghc ? nixpkgs.haskell.packages.ghc881.ghc, docs ? false }:
```

Comment:
Can we have this default to a pinned nixpkgs version which is shared with `release.nix` for CI? The version there is 20.03 so it would likely work.

Reply:
Yes, let's pin it.

Reply:
If I pin it, will it cause issues with the GitHub Ci?

Reply:
It shouldn't.

Reply:
We should pull the pin out of release.nix and use it here and there so they always use the same pin though.

Reply:
I guess that isn't strictly necessary, but if we actually can't use the same pin that would be a bit unfortunate.

Reply:
I want to make the docs build in CI, so ideally they can share the version. (We can of course update it if need be.)